lib/language_list.ex

```diff
@@ -0,0 +1,29 @@
+defmodule LanguageList do
+  @moduledoc false
+
+  def new do
+    []
+  end
+
+  def add(list, lang) do
+     [lang | list]
+  end
+
+  def remove(list) do
+    [_ | tail] = list
+    tail
```

Comment:
Pattern-match it directly in the function clause:
```suggestion
  def remove([_ | tail]) do
    tail
```